sprint-guide/index.njk

```diff
@@ -3,42 +3,45 @@ layout: 'layouts/base'
 ---
 
 <div id="home-page">
-  <div class="hero">
+  <div>
     <h1>
       <span class="eyebrow">
         Design Sprint Guide
       </span>
       <span class="headline | tbds-font-weight-bold">
-        Design, prototype & test ideas quickly.
+        Design, prototype & test ideas <span class="headline-underline">quickly.</span>
       </span>
     </h1>
   </div>
 
-  <section class="tbds-inline-stack tbds-inline-stack--gap-6" id="design-sprint">
-    <div class="tbds-inline-stack__item">
-      <div class="measure">
-        <h2>What is a design sprint?</h2>
-        <p>
-          A Product Design Sprint is a technique to quickly design,
-          prototype, and test the viability of an idea, product, or feature.
-          The design sprint consists of 5 phases (typically days),
-          starting with design thinking and ending with a user-tested prototype.
-          <a href="/introduction">Read more</a>
-        </p>
+  <section class="hero" id="design-sprint">
+    <div class="tbds-block-stack measure">
+      <div class="tbds-block-stack__item curvy-arrow">
+        <div>
+          <h2>What is a design sprint?</h2>
+          <p>
+            A Product Design Sprint is a technique to quickly design,
+            prototype, and test the viability of an idea, product, or feature.
+            The design sprint consists of 5 phases (typically days),
+            starting with design thinking and ending with a user-tested prototype.
+            <a href="/introduction" class="link--red">Read more</a>
```

Comment:
This link is broken in the preview deploy. At first I thought it was because it now needs to be `/sprint-guide/introduction` but it appears this page doesn't exist at all.

Reply:
I went ahead and removed it because, like you pointed out, I don't think we have a page for it anywhere.

Should I go ahead and change all the links to have `/sprint-guide/` prepended to them?

Reply:
Yeah, they'll need it.